helm/hpcc/values.yaml

```diff
@@ -167,6 +167,7 @@ eclccserver:
   replicas: 1
   ## Set to false if you want to launch each workunit compile in its own container, true to run as child processes in eclccserver pod.
   useChildProcesses: false
+  keepJobs: all
```

Comment:
Would be nice if there was a way to override this on all debug systems...

I suspect will get to the point where I want to maintain a separate "debugging" values.yaml and use that from startall.sh - the debugging values.yaml may or may not be something we will want to check in.

Reply:
I removed this line ad created a (private, unreleased) version of values for debugging purposes instead.